README.md

```diff
@@ -296,16 +296,17 @@ int6 = 5_349_221
 int7 = 1_2_3_4_5     # VALID but discouraged
 ```
 
-Leading zeros are not allowed. Hex, octal, and binary forms are not allowed.
-Values such as "infinity" and "not a number" that cannot be expressed as a
-series of digits are not allowed.
+Leading zeros are not allowed. Integer values `-0` and `+0` are valid and
+identical to an unprefixed zero. Hex, octal, and binary forms are not allowed.
 
 64 bit (signed long) range expected (−9,223,372,036,854,775,808 to
 9,223,372,036,854,775,807).
 
 Float
 -----
 
+Floats should be implemented as 64-bit IEEE 754 values.
```

Comment:
It should say IEEE 754 binary64 values, since there's a 64-bit decimal64 as well.

Reply:
Ah, fair enough, will change.